public/css/theme.css

```diff
@@ -1,6 +1,7 @@
 body {
     background-color: #0A0A0A;
     color: #E9E9E9;
+    font-family: Sans-serif, serif;
```

Comment:
```suggestion
    font-family: sans-serif, serif;
```
Lower case for consistency.

Reply:
Has to be captialized